docs/2-techdoc/62304/software-development-maintenance-plan.md

```diff
@@ -82,22 +85,31 @@ as software safety class A. The (external) risk control measures include:
 
 #### Programming Languages
 
+<!-- > List the languages you’ll be using, including compiler and language versions. -->
+
 | System                          | Name   | Version |
 | ------------------------------- | ------ | ------- |
 | Local DICOM server and ML model | Python | 3.9     |
```

Comment:
elsewhere you listed >=3.9, <3.12 for python... or something like that.  but here you only list 3.9

Reply:
For the actual released and regulated Medical Device binary for now, it will be 3.9. That is the version that will go through CI etc. In the future I expect I'll have a `propagate` like function that updates a macro within the docs to match what is being utilised within the regulated build.